includes/Classifai/Providers/Azure/Read.php

```diff
@@ -125,7 +125,20 @@ public function read_document() {
 
 		$filesize = filesize( get_attached_file( $this->attachment_id ) );
 		if ( ! $filesize || $filesize > computer_vision_max_filesize() ) {
-			return $this->log_error( new WP_Error( 'size_error', esc_html__( 'Document does not meet size requirements. Please ensure it is smaller than the maximum threshold (defaults to 4MB).', 'classifai' ), $metadata ), $attachment_id );
+			return $this->log_error(
+				new WP_Error(
+					'size_error',
+					esc_html(
+						sprintf(
+							// translators: %1$s is the document file size, %2$s is the current default max filesize
+							__( 'Document (%1$s) does not meet size requirements. Please ensure it is smaller than the maximum threshold (currently %1$s, defaults to 4MB).', 'classifai' ),
```

Comment:
`%1$s` is duplicated here. I think the second one should be `%2$s`

```suggestion
							__( 'Document (%1$s) does not meet size requirements. Please ensure it is smaller than the maximum threshold (currently %2$s, defaults to 4MB).', 'classifai' ),
```